cli/src/main.rs

```diff
@@ -1,12 +1,15 @@
-use std::path::PathBuf;
+use std::{io::Write, path::PathBuf};
 
 use clap::Parser;
+use color_eyre::Result;
+
+mod path;
 
 #[derive(Parser, Debug)]
 struct Args {
     /// Defines where your TS bindings will be saved by setting TS_RS_EXPORT_DIR
-    #[arg(long, short)]
-    output_directory: Option<PathBuf>,
+    #[arg(long, short, default_value = "./bindings")]
+    output_directory: PathBuf,
```

Comment:
I had to go back on obeying the environment variable if this argument isn't provided, otherwise there'd be no way to find the metadata file, as the CLI searches paths relative to the user's `pwd`.
Instead, we now always obey the cli (we could remove the default value to show the user that the CLI overrides the env variable)